### DISCUSSION TITLE
Customisable ruleset icon colours

### DISCUSSION THREAD

**Discussion by SherlockWh0** [ppy/osu#27203] [Ideas]
Currently, the osu! interface features these somewhat bland interface buttons:
<img width="333" alt="Screenshot 2024-02-17 at 15 39 10" src="https://github.com/ppy/osu/assets/107404453/68ec895f-ab3b-4852-bcc3-05c2333e688f">
The color for selection is a shade of green, and my idea is to add some colour to these icons

For Custom rulesets, these could be developer-assignable, but for the default rulesets, here's how I feel they should go:

osu!: Pink, like pippi
osu!taiko: Either Red or Green, for Mocha
osu!catch: Blue, like Yuzu
osu!mania: Purple, for Mani and Mari

**Comment by Ekischleki** [comment]
Seems like a good idea! I'll try implementing that. If you have a color palette (hex codes), for the standard rulesets, please tell me.

**Reply by Ekischleki** [reply]
#27207
